addon/components/sticky-element.js

```diff
@@ -276,6 +276,9 @@ export default Component.extend({
   },
 
   updatePosition() {
+    if(this.get('isDestroyed') || this.get('isDestroying')) {
```

Comment:
This function gets called in a few places, but I *think* the only one where #58 is caused is `_pollTaks()` (which is called with `Ember.run.later`, so would explain why this gets called when the component is already destroyed). Can you try to move the check there, so there is no performance cost for the other calls?